src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Link, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { fetchRegistration } from '../../../api/registrationApi';
+import { Registration, RegistrationStatus } from '../../../types/registration.types';
+
+interface DeletedRegistrationInformationProps {
+  registration: Registration;
+}
+
+export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+  const { t } = useTranslation();
+  const duplicateRegistrationQuery = useQuery({
+    enabled: !!registration.duplicateOf,
+    queryKey: ['registration', registration.duplicateOf],
+    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    meta: { errorMessage: t('feedback.error.get_registration') },
+  });
+
+  const duplicateRegistrationTitle =
+    duplicateRegistrationQuery.data?.entityDescription?.mainTitle ?? registration.duplicateOf;
+  const unpublishingNote = registration.publicationNotes?.find((note) => note.type === 'UnpublishingNote');
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '0.1rem',
+        marginTop: '1rem',
+      }}>
```

Comment:
flex som bare har et element trengs ikke :)
Men om det er lagt inn fordi det skal brukes i kommende PR(s) rundt hvem/når er det fint å ta det med nå da 👍 

```suggestion
```

Reply:
Er for kommende PR med bilde av person og dato.